pkg/vm/createCustom.go

```diff
@@ -6,18 +6,26 @@ import (
 	"os"
 
 	"github.com/ava-labs/avalanche-cli/cmd/prompts"
+	"github.com/ava-labs/avalanche-cli/pkg/app"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
 	"github.com/ava-labs/avalanche-cli/ux"
-	"github.com/ava-labs/avalanchego/utils/logging"
 )
 
-func CreateCustomGenesis(name string, log logging.Logger) ([]byte, error) {
+func CreateCustomGenesis(name string, app *app.Avalanche) ([]byte, *models.Sidecar, error) {
```

Comment:
I think app is not used in the function

Reply:
That is correct. But all the other similar functions take the app as a parameter, as it holds all shared information. It looked consistent to me.

Having said that, I can remove it if you insist.

Reply:
Fixed